src/array_api_stubs/_draft/array_object.py

```diff
@@ -343,9 +341,52 @@ def __dlpack__(
 
         Notes
         -----
+        Major DLPack versions represent ABI breaks, minor versions represent
+        ABI-compatible additions (e.g., new enum values for new data types or
+        device types).
+
+        The ``max_version`` keyword was introduced in v2023.12, and goes
+        together with the ``DLManagedTensorVersioned`` struct added in DLPack
+        1.0. This keyword may not be used by consumers for some time after
+        introduction. It is recommended to use this logic in the implementation
+        of ``__dlpack__``:
+
+        .. code:: python
+
+            if max_version is None:
+                # Keep and use the DLPack 0.X implementation
+                # Note: in >= 2 years from now (but ideally as late as
```

Comment:
I would replace ">= 2 years from now" with a specific date, as it won't be clear to the reader when this was written (either in the spec or for someone who's copied this to their codebase).